pkg/io/opener.go

```diff
@@ -17,106 +17,392 @@ limitations under the License.
 package io
```

Comment:
Sounds good to me

Reply:
I moved the package and restored the old io package. I just realized again that the old package is completely unused now, because the only components which did use it where tide, gerrit and status-reconciler. The other components which will be migrated in follow-up PRs aren't using the `io` or `io/v2` package yet.